tests/db/model/token/utils.js

```diff
@@ -63,6 +64,6 @@ module.exports = {
       });
       return tu.db.Token.bulkCreate(toCreate);
     })
-    .catch(console.error);
+    .catch(logger.error);
```

Comment:
probably don't need this

Reply:
So what to add in the catch? Just an empty function or remove the catch entirely?

Reply:
remove the catch entirely. the test failed.

Reply:
Fixed.